medusa/common.py

```diff
@@ -310,67 +308,62 @@ def scene_quality(name, anime=False):
             # BluRay
             if ep.bluray and (full_hd or hd_options):
                 result = Quality.FULLHDBLURAY if full_hd else Quality.HDBLURAY
-            # HD TV
-            elif not ep.bluray and (full_hd or hd_options):
+            # HDTV
+            elif full_hd or hd_options:
                 result = Quality.FULLHDTV if full_hd else Quality.HDTV
-            # SD DVD
+            # SDDVD
             elif ep.dvd:
                 result = Quality.SDDVD
-            # SD TV
+            # SDTV
             elif sd_options:
                 result = Quality.SDTV
 
             return Quality.UNKNOWN if result is None else result
 
         # Is it UHD?
-        if ep.vres in [2160, 4320] and ep.scan == 'p':
-            # BluRay
-            full_res = (ep.vres == 4320)
-            if ep.avc and ep.bluray:
```

Comment:
Just to be clear, removing the `ep.avc` tag checks means any video codec is accepted.
```py
avc = re.compile(r'([xh].?26[45]|(?:he|a)vc)', re.IGNORECASE)
```
https://regex101.com/r/e3tGNV/1/

Could you clarify which cases this change applies to?

Reply:
If I'm not mistaken it's intended.

Reply:
Previously the function was called `scene_quality` and in that case, it made sense to check for the codec, as it is required by scene standards to be present in the release name. My goal here was to make it more generic so that we have a quick way to determine the quality from the name. The reasoning behind it is: do we really need to know the codec to determine the quality from the name? I don't think we do and since most of the users are probably using non-scene namings (without codec or source, but only with resolution) it made sense to remove that check altogether.

Reply:
Ah, very well, makes sense.